packages/mds-schema-validators/validators.ts

```diff
@@ -118,7 +119,9 @@ export const policySchema = Joi.object().keys({
   end_date: Joi.date().timestamp('javascript').allow(null),
   prev_policies: Joi.array().items(Joi.string().guid()).allow(null),
   provider_ids: Joi.array().items(Joi.string().guid()).allow(null),
-  rules: Joi.array().min(1).items(ruleSchema).required()
+  rules: Joi.array().min(1).items(ruleSchema).required(),
+  rate_recurrence: Joi.string().valid('once', 'each_time_unit', 'per_complete_time_unit'),
```

Comment:
Can these values be made into a string literal with a const assertion?
```ts
export const RATE_RECURRENCE_VALUES = ['once', 'each_time_unit', 'per_complete_time_unit'] as const
export type RATE_RECURRENCE = typeof RATE_RECURRENCE_VALUES[number]
```
Then, you can change the policySchema to:
```suggestion
  rate_recurrence: Joi.string().valid(...RATE_RECURRENCE_VALUES),
```
You would also then gain the benefit of the string literal `RATE_RECURRENCE` type where you need it.